src/app/(proper_react)/redesign/(authenticated)/user/welcome/EnterInfo.module.scss

```diff
@@ -85,6 +85,23 @@
   @media screen and (max-width: $screen-lg) {
     grid-template-columns: 1fr;
   }
+
+  input {
+    min-height: 3rem;
+
+    &::placeholder {
+      // overwrite default opacity for placeholders in Firefox
+      opacity: 1;
```

Comment:
I don’t have strong feelings about this either. From a design and implementation perspective, I find it confusing that opacity is applied by default. I’d like to overwrite it here because otherwise in Firefox this is leading to the text inputs not having the same text color as for the date input.

<img width="679" alt="image" src="https://github.com/mozilla/blurts-server/assets/13835474/cebd69d7-136f-4bb1-9614-957e969ef4e7">

Reply:
Ha OK, yeah that sounds like a Firefox issue then. I guess maybe the reasoning is that in the DoB field it's not a placeholder, but sub-inputs that you can click? Either way fine to work around it here I suppose.

Reply:
But yeah, good point that it is probably there for a reason. I find it just really confusing that setting the placeholder explicitly to a color will not be shown as intended. I went ahead and merged the main issue can be fixed for now and hope it’s not too bad of a praxis.